core/tests/protractor_desktop/communityDashboard.js

```diff
@@ -43,3 +45,59 @@ describe('Community dashboard page', function() {
     general.checkForConsoleErrors([]);
   });
 });
+
+describe('Admin page community reviewer form', function() {
+  var adminPage = null;
+  var communityDashboardPage = null;
+
+  beforeAll(function() {
+    adminPage = new AdminPage.AdminPage();
+    communityDashboardPage = (
+      new CommunityDashboardPage.CommunityDashboardPage());
+    users.createUser('translator@community.com', 'translator');
+    users.createUser('voiceartist@community.com', 'voiceartist');
+    users.createUser('questionreviewer@community.com', 'questionreviewer');
+    users.createAdmin('management@adminTab.com', 'management');
```

Comment:
What do you think of storing these emails and usernames as variables? I think there are differing opinions on this style, so I don't have a strong preference here. The main benefit of variables that I see is it would help future developers catch bugs from typos.